src/main/java/org/skyscreamer/jsonassert/LocationAwareValueMatcher.java

```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright 2012 Duncan Mackinder.
```

Comment:
Sorry i can't change this header in a short time (at least one month)
may change you?
listing at least those who, by heart, helped. 
unless the file in question was not taken from another project
regards